modules/external-api/src/main/java/org/opencastproject/external/common/ApiVersion.java

```diff
@@ -20,28 +20,34 @@
  */
 package org.opencastproject.external.common;
 
-/** The existing versions */
+/** The existing External API versions */
 public enum ApiVersion {
 
-  VERSION_UNDEFINED("v*.*.*"),
+  VERSION_1_0_0(1, 0, 0),
+  VERSION_1_1_0(1, 1, 0);
 
-  VERSION_1_0_0("v1.0.0");
+  /** The most recent version of the External API */
+  public static final ApiVersion CURRENT_VERSION = VERSION_1_1_0;
 
-  public static final ApiVersion CURRENT_VERSION = VERSION_1_0_0;
+  private int major;
+  private int minor;
+  private int patch;
```

Comment:
We had quite some discussions about how we proceed considering versioning as the versioning was not working in means of we could not return the API version in the responses (which again the specification says to do so).

Finally, we found that it can be fixed quite easily.

Except of this, this PR is not about API versioning at all and does not attempt to change it.